Lib/typing.py

```diff
@@ -1396,8 +1396,15 @@ def _get_protocol_attrs(cls):
 
 
 def _is_callable_members_only(cls):
+    attr_names = _get_protocol_attrs(cls)
+    annotations = getattr(cls, '__annotations__', {})
     # PEP 544 prohibits using issubclass() with protocols that have non-method members.
-    return all(callable(getattr(cls, attr, None)) for attr in _get_protocol_attrs(cls))
+    for attr_name in attr_names:
+        attr = getattr(cls, attr_name, None)
+        if not (callable(attr)
+            or (getattr(annotations.get(attr_name), '__name__', None) == 'ClassVar')):
```

Comment:
Should we also check that the module name of object is `typing`?

```suggestion
        ann = annotations.get(attr_name)
        if not (callable(attr)
            or (getattr(ann, '__name__', None) == 'ClassVar') and getattr(ann, '__module__', None) == 'typing'):
```

Reply:
I think we don't need to be so strict. Also it could be imported from `typing_extensions` which would make it fail incorrectly.